go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	golang.org/x/sys v0.1.0
 	golang.org/x/text v0.4.0
 	gopkg.in/sohlich/elogrus.v3 v3.0.0-20180410122755-1fa29e2f2009
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
was it moved there automatically?

Reply:
Yeah presumably `go mod tidy` since I definitely didn't move it manually.